subprojects/snapshots/src/main/java/org/gradle/internal/vfs/watch/UnableToStartWatchingException.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.vfs.watch;
+
+public class UnableToStartWatchingException extends RuntimeException {
```

Comment:
I think this would make sense to have as a checked exception. It's good to see where it can come from. WDYT?

Reply:
We don't use checked exceptions in the Gradle code base and I think that is good. The API declares that it throws this exception and that is good enough.

Reply:
Let's agree to disagree here.